publiccode.yml

```diff
@@ -0,0 +1,73 @@
+publiccodeYmlVersion: '0.2'
+name: Sala Operativa VVF
+releaseDate: '2018-05-17'
+url: 'https://github.com/vvfosprojects/sovvf'
+developmentStatus: stable
+softwareType: standalone/web
+platforms:
+  - web
+usedBy:
+  - Corpo Nazionale dei Vigili del Fuoco
+categories:
+  - fleet-management
+  - data-collection
+  - data-analytics
+  - data-visualization
+maintenance:
+  type: internal
+  contacts:
+    - name: ??? 
+      email: ??? 
+      affiliation: Corpo Nazionale dei Vigili del Fuoco
+legal:
+  license: AGPL-3.0-or-later
+  mainCopyrightOwner: 2018 Corpo Nazionale dei Vigili del Fuoco
+  repoOwner: Corpo Nazionale dei Vigili del Fuoco
+intendedAudience:
+  scope:
+    - emergency-services
+    - defence
+    - security
+    - transportation
+  countries:
+    - it
+localisation:
+  localisationReady: false
+  availableLanguages:
+    - it
+it:
+  countryExtensionVersion: '0.2'
+  conforme:
+    misureMinimeSicurezza: true
+    gdpr: true
+  riuso:
+    codiceIPA: m_it
+description:
+  en:
+    localisedName: Sala Operativa VVF
+    genericName: sovvf
+    shortDescription: Sala Operativa VVF
+    longDescription: >
```

Comment:
SOVVF è il progetto di reingegnerizzazione del software applicativo per le sale operative del Corpo Nazionale dei Vigili del Fuoco. Il software consente l'acquisizione delle chiamate di soccorso, la gestione delle code, il dispiegamento delle risorse per il contrasto all'emergenza, la pianificazione anticipata del dispositivo di soccorso. Il sistema è costituito da un moduilo di backend sviluppato in tecnologia .NET Core WebApi e un frontend sviluppato in tecnologia Angular7, oltre ad altri moduli a corredo per la gestione di processi collaterali. Il software è al momento in piena fase di sviluppo e non è acquisibile per scenari di esercizio.

Reply:
Added :+1: